docs/source/cpp/parquet.rst

```diff
@@ -571,6 +571,19 @@ More specifically, Parquet C++ supports:
 * EncryptionWithFooterKey and EncryptionWithColumnKey modes.
 * Encrypted Footer and Plaintext Footer modes.
 
+Field Id
+----------
+
+The parquet format supports an optional integer field_id which can be assigned
+to a field. This is used in the `iceberg specification <https://github.com/apache/iceberg/blob/main/format/spec.md#column-projection>` __
```

Comment:
I'd suggest not mentioning iceberg to be neutral.

Reply:
I mention it because it's used there. Any better links could also be ok

Reply:
I find it useful to give context, so perhaps the wording can be improved to make it clear that it's just an example?